server/wfnews-persistence/src/main/resources/ca/bc/gov/nrs/wfnews/persistence/v1/dao/mybatis/mapper/AttachmentMapper.xml

```diff
@@ -35,7 +38,7 @@
             uploaded_by_user_guid, uploaded_by_timestamp, revision_count, create_user, create_date, update_user,
             update_date, primary_ind)
 	  VALUES 
-           (#{dto.attachmentGuid,    javaType=java.lang.String,  jdbcType=BINARY,  mode=IN},
+           (#{fileAttachmentGuid,    javaType=java.lang.String,  jdbcType=BINARY,  mode=IN},
```

Comment:
Wait, I thought we wanted to use the original attachment guid? So we had a simple mapping back to the source attachment for updates from the sync?

Reply:
Forgot about that, I'll change it back